conanfile.py

```diff
@@ -0,0 +1,39 @@
+from conans import ConanFile, CMake, tools
+
+
+class HelloConan(ConanFile):
+    name = "LibConfig"
+    version = "1.7.2"
+    license = "GNU LESSER GENERAL PUBLIC LICENSE"
+    author = """Mark Lindner - Lead developer & maintainer.
+                Daniel Marjamäki - Enhancements & bugfixes.
+                Andrew Tytula - Windows port.
+                Glenn Herteg - Enhancements, bugfixes, documentation corrections.s
+                Matt Renaud - Enhancements & bugfixes.
+                JoseLuis Tallon - Enhancements & bugfixes"""
+    url = "hyperrealm.github.io/libconfig/"
+    description = "Libconfig is a simple library for processing structured configuration files, like this one. This file format is more compact and more readable than XML. And unlike XML, it is type-aware, so it is not necessary to do string parsing in application code."
+    topics = ("config")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": False}
+    generators = "cmake"
+    exports_sources="*"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        self.copy("*.h", dst="include", src="lib")
+        self.copy("*.hpp", dst="include", src="lib")
+        self.copy("*.h++", dst="include", src="lib")
+        self.copy("*.dll", dst="bin", keep_path=False)
+        self.copy("*.so", dst="lib", keep_path=False)
+        self.copy("*.dylib", dst="lib", keep_path=False)
+        self.copy("*.a", dst="lib", keep_path=False)
+
+    def package_info(self):
+        self.cpp_info.libs = ["config", "config++"]
```

Comment:
conan does not (yet) support creating multiple FindXXX.cmake scripts.

```suggestion
        # FIXME: `libconfig` is `libconfig::libconfig` in `libconfigConfig.cmake`
        # FIXME: `libconfig++` is `libconfig::libconfig++` in `libconfig++Config.cmake`
        self.cpp_info.components["libconfig_c"].libs = ["libconfig"]
        self.cpp_info.components["libconfig_c"].names["cmake_find_package"] = ["libconfig"]
        self.cpp_info.components["libconfig_c"].names["cmake_find_package_multi"] = ["libconfig"]
        self.cpp_info.components["libconfig_c"].names["pkg_config"] = "libconfig"

        self.cpp_info.components["libconfig_cpp"].libs = ["libconfig++"]
        self.cpp_info.components["libconfig_cpp"].names["cmake_find_package"] = ["libconfig++"]
        self.cpp_info.components["libconfig_cpp"].names["cmake_find_package_multi"] = ["libconfig++"]
        self.cpp_info.components["libconfig_cpp"].names["pg_config"] = "libconfig++"
```

Reply:
I've tried creating a minimum running example: 

```
cmake_minimum_required(VERSION 3.17)
project(libconfigconantest)

set(CMAKE_CXX_STANDARD 14)

# include conan libraries
include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
conan_basic_setup()

# include the library headers created by conan
include_directories(${CONAN_INCLUDE_DIRS})

add_executable(libconfigconantest main.cpp)
target_link_libraries(libconfigconantest ${CONAN_LIBS})
```

This results in: 

```
/home/lukas/.local/share/JetBrains/Toolbox/apps/CLion/ch-0/203.6682.181/bin/cmake/linux/bin/cmake --build /home/lukas/CLionProjects/libconfigconantest/build --target libconfigconantest -- -j 4
[ 50%] Building CXX object CMakeFiles/libconfigconantest.dir/main.cpp.o
[100%] Linking CXX executable bin/libconfigconantest
/usr/bin/ld: cannot find -llibconfig
/usr/bin/ld: note to link with /home/lukas/.conan/data/libconfig/1.7.2/_/_/package/73468d6cd433312abe58cd86e0da983f5d6f13ce/lib/libconfig.a use -l:libconfig.a or rename it to liblibconfig.a
/usr/bin/ld: cannot find -llibconfig++
/usr/bin/ld: note to link with /home/lukas/.conan/data/libconfig/1.7.2/_/_/package/73468d6cd433312abe58cd86e0da983f5d6f13ce/lib/libconfig++.a use -l:libconfig++.a or rename it to liblibconfig++.a
collect2: Fehler: ld gab 1 als Ende-Status zurück
gmake[3]: *** [CMakeFiles/libconfigconantest.dir/build.make:104: bin/libconfigconantest] Fehler 1
gmake[2]: *** [CMakeFiles/Makefile2:96: CMakeFiles/libconfigconantest.dir/all] Fehler 2
gmake[1]: *** [CMakeFiles/Makefile2:103: CMakeFiles/libconfigconantest.dir/rule] Fehler 2
gmake: *** [Makefile:138: libconfigconantest] Fehler 2
```

There is still something wrong with the created build targets. Any suggestions? Or is my Cmake faulty?